src/vm/runners/builtin_runner/range_check.rs

```diff
@@ -15,31 +15,29 @@ use crate::vm::vm_core::VirtualMachine;
 use crate::vm::vm_memory::memory::{Memory, ValidationRule};
 use crate::vm::vm_memory::memory_segments::MemorySegmentManager;
 
-use super::BuiltinRunner;
-
 pub struct RangeCheckBuiltinRunner {
     ratio: u32,
     base: isize,
     stop_ptr: Option<usize>,
     pub(crate) cells_per_instance: u32,
     pub(crate) n_input_cells: u32,
-    inner_rc_bound: BigInt,
+    inner_rc_bound: isize,
```

Comment:
This value is constant and always positive, we can use usize instead
```suggestion
    inner_rc_bound: usize,
```